elastic/kibana
# [Lens] Fixes the composite runtime fields description

## Summary

Closes https://github.com/elastic/kibana/issues/126312.

This bug appeared when we merged this [PR](https://github.com/elastic/kibana/pull/125183).

The field description was misleading.

![image](https://user-images.githubusercontent.com/17003240/157698018-2b006547-f7fb-463b-bc3e-651700b912ba.png)

With this fix, it works fine now. 

<img width="435" alt="image" src="https://user-images.githubusercontent.com/17003240/157685213-6a1befb2-d2cb-463d-b645-f9185cd9a0ad.png">

### How to test it
Follow the steps described [here](https://github.com/elastic/kibana/blob/6627bd8b3ab49415e397495d8e076f9861fcc894/x-pack/plugins/lens/server/routes/field_stats.ts#L83) to add a composite runtime field. Check the field stats.
